tests/testthat/test-scales.R

```diff
@@ -127,7 +131,7 @@ test_that("oob affects position values", {
 
   # Bars depend on limits and oob
   expect_equal(low_censor[[1]]$y, c(0.2, 1))
-  expect_equal(mid_censor[[1]]$y, c(0.5))
+  expect_equal(mid_censor[[1]]$y, numeric(0))
```

Comment:
I think previous test wasn't representative of what happens in a plot.
Because all bars have ymin = 0, and get censored at `c(3, 7)` limits, none of the bars are retained.